lib/feature/profile/manage_seeds_accounts/widgets/export_seed_sheet/export_seed_state.dart

```diff
@@ -0,0 +1,13 @@
+part of 'export_seed_cubit.dart';
+
+@freezed
+class ExportSeedState with _$ExportSeedState {
+  const factory ExportSeedState.initial() = _Initial;
+
+  const factory ExportSeedState.loading() = _Loading;
+
+  /// Error with [code] that should be get from localization.
+  const factory ExportSeedState.error(String code) = _Error;
```

Comment:
Why not use the translated (ready) string directly?

Reply:
-